config/rbac/dedicated-admin-role.yaml

```diff
@@ -0,0 +1,24 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: df-dedicated-admin-role
+  # aggregationRule will append below rules to dedicated-admins-cluster role if labels matchExpressions is valid
+  # aggregationRule : https://github.com/openshift/managed-cluster-config/blob/f4537242f9c54ee81bd049c458f3dc1e7ed9f838/deploy/rbac-permissions-operator-config/03-dedicated-admins-cluster.ClusterRole.yaml
+  # https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles
```

Comment:
looks like aggregation rules will not work now: https://issues.redhat.com/browse/MTSRE-581?focusedCommentId=20553199&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-20553199
SRE-P team wants it to be namespace scope Role instead of cluster-wide. So, we need to create both Role/RoleBinding now for the dedicated-admins user.

Reply:
@SanjalKatiyar  yes, I did the changes and code commited